src/console/index.js

```diff
@@ -0,0 +1,60 @@
+
+import Replacer from '../replacer';
+
+export default class ConsoleTransport {
+    constructor (replacer = createReplacer()) {
+        this.replacePlaceholders = false;
+        this._replacer = replacer;
+        this._console = console ? console : createFakeConsole();
+    }
+
+    write (method, formattedArgs) {
+        if (!this._console[method]) {
+            return;
```

Comment:
It's possible, that just one transport would support particular method, and others would ignore it. On the other hand, some transports doesn't even look at `method` value, like `WebFileTransport`